src/esp/assets/GenericSemanticMeshData.cpp

```diff
@@ -210,6 +227,15 @@ GenericSemanticMeshData::buildSemanticMeshData(
           // color is found in ssd mapping, so is legal color
           semanticID = ssdColorToIDAndRegionIter->second.first;
           regionID = ssdColorToIDAndRegionIter->second.second;
+
+          // TEMP FOR VERT-BASED OBB CALC
+          // aggregate
+          auto vert = semanticData->cpu_vbo_[vertIdx];
+          vertAggregate[semanticID] += vert;
+          vertMax[semanticID] = vertMax[semanticID].cwiseMax(vert);
+          vertMin[semanticID] = vertMin[semanticID].cwiseMin(vert);
```

Comment:
If you'd use Magnum vectors, there's [Math::minmax()](https://doc.magnum.graphics/magnum/namespaceMagnum_1_1Math.html#a6d0a7d68d7aae97ad18ebdd1d9ee66e5) that calculates both bounds at the same time, saving quite a few comparison ops.

Reply:
Only didn't do that because source and dest had to be eigen vecs.  Would certainly prefer to use magnum.